files/image_config/monit/memory_checker

```diff
@@ -96,6 +96,19 @@ def check_memory_usage(container_name, threshold_value):
         sys.exit(4)
 
 
+def is_service_active(service):
+    """Test if service is running.
+
+    Args:
+        command: A string contains the service name
```

Comment:
`command` should be `service_name`, right?

Reply:
Done.